### ISSUE TITLE
Accessibility features

### ISSUE THREAD

**Issue by irmozius** [irmozius/burnt-steel-series#16]
Accessibility is very important and should be required in all our games. Here are some guidelines to inform our choices.

Here are some i think must be implemented:

- Separate volume controls for different types of sound (music/dialogue/sfx)
  
- Captions for all voicelines, with UI customization
  
- Control remapping
